Misc/NEWS.d/next/Library/2021-11-10-00-22-14.bpo-45775.e5tVoB.rst

```diff
@@ -0,0 +1 @@
+Added implementions of colorsys.rgb_to_yuv and colorsys.yuv_to_rgb using the ATSC BT.709 constants
```

Comment:
```suggestion
Added implementions of :func:`colorsys.rgb_to_yuv` and :func:`colorsys.yuv_to_rgb` using the ATSC BT.709 constants.
```

Reply:
Updated, was using blurb-it so didn't know this was RST.